packages/apps/dev-wallet/src/pages/create-profile/create-profile.tsx

```diff
@@ -107,9 +105,23 @@ export function CreateProfile() {
             />
           </Stack>
           <Stack flexDirection="column">
-            <Button type="submit" isDisabled={!isValid}>
-              Continue
-            </Button>
+            {profileList.length ? (
+              <Button isDisabled={!isValid}>
+                <Link
+                  to={'/personalize-profile'}
```

Comment:
Shouldn't it be part of the `create` of handlesubmit function instead of the link in the button? 
Also currently the `Continue` button doesn't work on the first click while inputs of the password and confirm password are valid.

Reply:
If we skip the '/personalize-profile' page, then yes, it is better to do it in the script. Will change.

I put a link before because there is a good practice to use links, not buttons, when we change the URL.

Reply:
Using a button as a link can cause issues for users in terms of the accessibility (by navigating with keyboard or who use a screen reader)

Reply:
This does not make the whole button a link just the children is a link; its why the button does not work as 
expected 
also we don't need a separate page, instead you can use the Wizard (stepper) component to handel this.
which lets us to navigate between steps easier since the state is all in one place
check the signature-builder page.

Reply:
@javadkh2 the stepper is not meant to allow click to navigate in the top, only next and back, but indeed each step has the continue and back

Reply:
I changed the code.
No stepper for Personalize because it's not a step.